set.mm

```diff
@@ -646216,6 +646216,13 @@ D Fn ( I ... ( M - 1 ) ) /\
     ( cvv cv wnel cab wcel vex elirrv nelir 2th abbi2i eqcomi ) BACZMDZAENABMBF
     NAGMMAHIJKL $.
 
+  $( Alternative to ~ wcdeq and ~ df-cdeq .  This flattens the syntax
+     representation ( wi ( weq vx vy ) wph ) to ( wcdeqALT vx vy wph ),
+     accomplishing the stated purpose of ~ df-cleq .  (Contributed by SN,
```

Comment:
That does not really accomplish the purpose of df-cdeq (BTW, you wrote "df-cleq"). You could say it "illustrates the comment of df-cdeq". Overall, I find this theorem of dubious interest.

Reply:
It accomplishes what df-cdeq says it accomplishes: flattening the syntax tree

Reply:
In retrospect I really misunderstood the suggestion, sorry about that. I changed "stated" to "commented" to make it more clear, is it okay now? @benjub

Reply:
I think "illustrating the comment of" is more accurate than "accomplishing the commented purpose of". Actually, the latter sounds weird in English, but I'm no native speaker.

Since it's inside your mathbox, this conversation is non-blocking.